docs/zh/guides/advanced/database/field/basic/percent.md

```
@@ -1 +1,11 @@
-# 百分比
\ No newline at end of file
+# 百分比
+
+### 介绍
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)